src/openstudio_lib/SimSettingsView.cpp

```diff
@@ -140,6 +140,9 @@ SimSettingsView::SimSettingsView(bool isIP,
   m_loadsConvergenceToleranceValue(nullptr),
   m_temperatureConvergenceToleranceValue(nullptr),
   m_solarDistribution(nullptr),
+  m_doHVACSizingSimulationforSizingPeriods(nullptr),
+  m_maximumNumberofHVACSizingSimulationPasses(nullptr),
```

Comment:
Ctor initialize new fields in the right place to match hpp

Reply:
We should turn on static analysis to check this, http://cppcheck.sourceforge.net/